asciinema/task.yaml

```diff
@@ -1,4 +1,4 @@
-apiVersion: tekton.dev/v1alpha1
+apiVersion: tekton.dev/v1beta1
```

Comment:
in v1beta1 `spec.inputs` was split into `spec.params` and `spec.resources.inputs`

Reply:
That actually explains an issues I was having with some demo code I was trying to run. 

I will do a new version where I only use `spec.resources.inputs`.Adding three more lines for `spec.params` will make it so that the `task.yaml` will scroll out of the video frame when the command `cat task.yaml` is executed.